virttest/qemu_devices/qdevices.py

```diff
@@ -1877,15 +1877,13 @@ def start_daemon(self):
         fsd_cmd = '%s --socket-path=%s' % (self.get_param('binary'),
                                            self.get_param('sock_path'))
         fsd_cmd += ' -o source=%s' % self.get_param('source')
-        fsd_cmd += ' -d'
```

Comment:
Suggest making whether using the daemon mode configurable.

Reply:
Yes, make sense, Could we implement it through setting `self.get_param('extra_options')`, line 1882 ~ 1883 ?

Reply:
Looks possible, but just not that formal. Anyway, it's acceptable.

Reply:
or could we check whether the call use `debug` mode then whether to execute the previous steps, for instance:
```
def __init__(self, aobject, binary, sock_path, source, extra_options=None, debug_mode=False):
    ...
    fsd_cmd += ' -o source=%s' % self.get_param('source')
    if debug_mode:
        fsd_cmd += ' -d'
    ...
    ...
    if debug_mode
        self.set_param('status_active', 'Waiting for vhost-user socket connection')
```

Reply:
@YongxueHong yes, that would be an possible approach.

Reply:
@luckyh 
Thanks, How about you? @zhenyzha

Reply:
@luckyh @YongxueHong It's ok for me, update later.